invenio_app_rdm/theme/views.py

```diff
@@ -35,11 +35,7 @@ def create_blueprint(app):
     @blueprint.before_app_first_request
     def init_menu():
         """Initialize menu before first request."""
-        item = current_menu.submenu("main.deposit")
-        item.register(
-            "invenio_app_rdm_records.deposit_search", _("Uploads"), order=1)
-
-        item = current_menu.submenu('plus.deposit').register(
+        current_menu.submenu('plus.deposit').register(
```

Comment:
are we sure that we want to remove this from here and hardcode in in the template?

ping @zzacharo to have another opinion

Reply:
What i deleted here is just the upload button that was on the left main nav portion, as was suggested in the UX. But i agree as discussed earlier that i can make a new submenu to make it dynamic